src/control/server/instance_storage_rpc.go

```diff
@@ -90,8 +90,10 @@ func (ei *EngineInstance) bdevFormat(p *bdev.Provider) (results proto.NvmeContro
 		engineIdx, cfg.Class, cfg.DeviceList)
 
 	res, err := p.Format(bdev.FormatRequest{
+		OutputPath: cfg.OutputPath,
 		Class:      cfg.Class,
 		DeviceList: cfg.DeviceList,
+		FileSize:   cfg.FileSize,
```

Comment:
Might be better to call these `ConfigPath` and `DevFileSize` or something like that, to indicate that they're related to the spdk configuration.